### PR TITLE
feat: make music label more dynamically configurable

### PR DESCRIPTION
**Pull request by rubiin**
This lets users specify how they want to display media label on bar

image

image

### CODE REVIEW

**Comment by Jas-SinghFSU** [comment]
Yeah... this is sick!

**Comment by rubiin** [comment]
I have also added some player icons for bar (vlc and mpv)
image

Also ability to disbale particular  media section with media menu setting

image

File: modules/menus/media/components/index.ts
Comment (Jas-SinghFSU) [inline modules/menus/media/components/index.ts:30]: Not a fan of this change. Something about the bar below the controls doesn't look appealing.

**Comment by Jas-SinghFSU** [comment]
> I have also added some player icons for bar (vlc and mpv) ![image](https://private-user-
> 
> Also ability to disbale particular optional media section with media menu setting
> 

Nice! Love the toggles. I have made changes to the logic to your tag logic. What I found was that if you have the current default format that it will add a dash to the front of the label even if you don't have an author. So when listening to a podcast which may not have the 'artist' field, it looks like ` - Some podcast name`.

I have made a new branch that lets you define conditional formatting via `{artist: - }{title}` where the item after the colon  is a string that will display ONLY if the artist or any value that the tag represents, exists.

So you can do `{artist: - }{title}` which will only display the ` - ` if the artist exists.

Feel free to merge with branch `media_improvements` to consume these changes.

**Comment by Jas-SinghFSU** [comment]
Looks good. Once you're able to merge my 'media_improvements' branch into this pr we should be good to merge.

**Comment by rubiin** [comment]
Done and dusted

File: options.ts
Comment (Jas-SinghFSU) [inline options.ts:936]:
```suggestion
            format: opt('{artist: - }{title}'),
```

File: widget/settings/pages/config/bar/index.ts
Comment (Jas-SinghFSU) [inline widget/settings/pages/config/bar/index.ts:727]:
```suggestion
                    title: 'Label Format',
```

**Comment by Jas-SinghFSU** [comment]
@rubiin Just need to resolve merge conflicts and we're good to merge after.

**Review by Jas-SinghFSU — APPROVED** [review]

**Pull request merged**
